web/src/app/users/UserContactMethodList.tsx

```diff
@@ -148,6 +149,9 @@ export default function UserContactMethodList(
         </React.Fragment>
       )
     }
+    if (cm.pending) {
+      return `${cm.formattedValue} - this contact method will be automatically deleted if not verified`
+    }
```

Comment:
This warning also needs to be added for unverified webhook contact methods

Reply:
Makes sense, added.